hack/install-deps.sh

```diff
@@ -27,6 +27,8 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+source $(dirname "${BASH_SOURCE[0]}")/utils.sh
```

Comment:
I think we can either source this into `hack/versions` and probably renaming to `hack/versions.sh`, or move `hack/versions` into `utils.sh`.
It is weird that we source a dependency of `hack/versions` here.

Reply:
yeah I moved what was left of hack versions into utils.sh.. so can pick the up with my pr

Reply:
Just found that test hasn't run. Will merge after test passes.

Reply:
@mikebrow Merged. Can you address this in your PR?

Reply:
absolutely!